suite-native/module-send/src/components/FeesFooter.tsx

```diff
@@ -0,0 +1,79 @@
+import { useContext } from 'react';
+
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { Text, Button, Box, Card, HStack, VStack } from '@suite-native/atoms';
+import { CryptoToFiatAmountFormatter, CryptoAmountFormatter } from '@suite-native/formatters';
+import { FormContext } from '@suite-native/forms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Translation } from '@suite-native/intl';
+
+type FeesFooterProps = {
+    onSubmit: () => void;
+    networkSymbol: NetworkSymbol;
+    totalAmount: string;
+};
+
+const footerWrapperStyle = prepareNativeStyle(utils => ({
+    marginBottom: utils.spacings.small,
+}));
+
+const cardStyle = prepareNativeStyle(utils => ({
+    paddingTop: utils.spacings.small,
+    paddingBottom: utils.spacings.extraLarge + utils.spacings.small,
```

Comment:
nit: I would prefer not to compose arbitrary numbers from predefined spacings, wdyt?

Reply:
7f3034dd1d